Tests/Clappr_Tests/BaseObjectTests.swift

```diff
@@ -70,6 +70,19 @@ class BaseObjectTests: QuickSpec {
 
                     expect(callbackWasCalled) == false
                 }
+
+                it("protect the main thread when plugin crashes in render") {
+                    let expectation = QuickSpec.current.expectation(description: "doesn't crash")
+                    let baseObject = BaseObject()
+
+                    baseObject.on(eventName) { _ in
```

Comment:
Should we move the `baseObject.on(eventName) { _ in ... }` one line up with the rest of the `Arrange` phase?
It looks like part of it, please correct me if I'm wrong.

Reply:
I will move it, thanks!